lte/gateway/deploy/roles/magma/files/envoy.yaml

```diff
@@ -0,0 +1,40 @@
+admin:
```

Comment:
can you replace this with `lte/gateway/python/magma/pipelined/tests/envoy-tests/envoy-cntr-prod.yaml`

Reply:
hmm thats where I copied it from